Client/Frontend/Browser/TabButtons.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  TabButtons.swift
+//  Client
+//
+//  Created by Tyler Lacroix on 6/27/16.
+//  Copyright © 2016 Mozilla. All rights reserved.
+//
```

Comment:
Also I think the name of this file is a bit confusing since we also have `TabsButton`. Maybe rename this to TabTrayButtonExtensions and move to the Widgets folder.